json-rpc/src/runtime.rs

```diff
@@ -19,6 +19,14 @@ use warp::{
     Filter,
 };
 
+// Counter labels for runtime metrics
+const LABEL_FAIL: &str = "fail";
```

Comment:
maybe a more descriptive error :P ?

Reply:
I'm gonna punt on this for now. I'm not sure how much ecosystem support there already is for the JSON RPC metrics, so will refrain from modifying the counter name. I agree though.. it's not great!